.github/ISSUE_TEMPLATE.md

```diff
@@ -1,26 +1,38 @@
+<!-- 
+Note:
+
 Please check the existing issues to make sure you're not duplicating a report.
 
 For bug reports, please provide the following information:
 
-In a terminal, run `zmanage info` and paste the output below:
+In a terminal, run `zmanage info` and paste the output below: -->
 
+#### Details:
```

Comment:
Maybe the sentence "In a terminal, ... and paste the output below:" could be changed to a comment inside a header like

    #### Output of `zmanage info`:
      ```
      #In a terminal, run `zmanage info` and paste the output here
      ```

And then there would be no `#### Details` header.